DPKI_operations.md

```diff
@@ -0,0 +1,38 @@
+# Chapter on PKI/DPKI operations
+
+
+## Introduction to PKI operations:
+In the public key infrastructure (PKI), there are two main operations, which are: **signing request**, and **revocation request**. 
+There is also another operation that is nothing more than a composition of revocation and signing, namely updating.
+
+## Certificate signing request (CSR):
+A CSR is a message from an entity to a trusted authority of the PKI, which contains information about the entity's identity and it is called *certification request information*. Specifically, it contains the name of the entity, the public key of the entity, and optionally a set of attributes. All of that information is signed with the private key of the entity.
+
+A certification request consists of three parts: 
+- certification request information. 
+- a signature algorithm identifier.
+- a digital signature on the certification request information.
+
+
+If everything is correct, the trusted authority will sign the certification request and return an X.509 certificate. This certificate contains the public key of the entity, a serial number, a validity period, the signature algorithm, and other information. [[1](#references)]
+
+## Certificate Revocation List (CRL):
+When a certificate expires or is revoked, the trusted authority will create a CRL. This CRL contains a list of revoked certificates, who made the revocation, and other information. It is signed with the private key of the trusted authority and it is freely available in a public repository. Each revoked certificate is identified in a CRL by its certificate serial number. 
```

Comment:
As far as I know, the CRL exists already and it is not created when a certificate is revoked. 
In the case of a revocation, the certificate is added to the CRL.
Moreover, expired certificates are not added to the CRL.

Reply:
yes, you're right. I am fixing it this right now.